platform_umbrella/apps/common_core/lib/common_core/notebooks/jupyter_lab_notebook.ex

```diff
@@ -82,6 +82,8 @@ defmodule CommonCore.Notebooks.JupyterLabNotebook do
     field :memory_requested, :integer
     field :memory_limits, :integer
 
+    field :node_type, Ecto.Enum, values: ~w(default nvidia_gpu)a, default: :default
```

Comment:
How do we like this for the field? How do we want to expose this in the UI? Just a dropdown?

Reply:
SHould they choose which GPU they want here? In that case then yeah it will be a drop down. If it's just on or off then I would make this boolean and use a switch.